Tasks/Gradle/task.json

```diff
@@ -55,6 +55,15 @@
             "required": true,
             "helpMarkDown": "Relative path from the repository root to the Gradle Wrapper script."
         },
+        {
+            "name": "cwd",
```

Comment:
Moving workingDir to be right below gradle wrapper input instead of being hidden in advanced section